docs/user_guide/monitoring.md

```diff
@@ -0,0 +1,114 @@
+# Monitor Dragonfly with Prometheus
+
+Currently metrics become an important part of observability. As for monitoring Dragonfly, we recommend you to use Prometheus.
+
+In Dragonfly project, there are two long-running process: supernode and dfdaemon. Both of these two components expose its metrics via `/metrics` endpoint, so Prometheus can get metrics from these two components. We will also support dfget metrics in the future. As for current metrics, you can check out this docs.
```

Comment:
s/process/processes.
s/Both of these two components expose/Each of the components exposes/g
s/these two components/each component

These is used to describe instances which are more than two.

Reply:
Done